src/hr/entities/hr.entity.ts

```diff
@@ -0,0 +1,31 @@
+import { BaseEntity, Column, Entity, JoinColumn, OneToOne, PrimaryGeneratedColumn } from 'typeorm';
+
+import { HrEntity } from '../../types';
+import { User } from '../../users/entities/user.entity';
+
+@Entity('Hr')
+export class Hr extends BaseEntity implements HrEntity {
```

Comment:
Warto może dodać datę dodania, datę aktualizacji ? Ale to może później, jak będziemy robić refaktor